eureka_ml_insights/metrics/metrics_base.py

```diff
@@ -144,13 +144,39 @@ def __evaluate__(self, answer_text, target_text, is_valid):
         else:
             return "incorrect"
 
+class ExactOrMatch(ExactMatch):
+    """This class checks for a case-insensitive, but otherwise exact match, and returns the or of them."""
```

Comment:
Since the CaseInsensitive class handles lower case transform, this function is doing exact match (case sensitive) I think.. if so maybe we update the comment here

Reply:
yes, will do